Docker/README.md

```diff
@@ -0,0 +1,70 @@
+
+# Integration rextporter and skycoin using Docker
+
+## Dockerfile for build rextporter image
+```docker
+FROM golang:alpine as builder
+RUN apk add git
+RUN mkdir -p $GOPATH/src/github.com/simelo
+RUN cd $GOPATH/src/github.com/simelo && git clone https://github.com/simelo/rextporter.git
+RUN cd $GOPATH/src/github.com/simelo/rextporter/cmd/rextporter && go install ./...
+
+FROM alpine:latest as final_layer
+WORKDIR /bin
+COPY --from=build_layer /go/bin/rextporter .
+```
+## Docker-compose
+```yaml
+version: '3.3'
+services:
+  
+  rextporter:
+    image: rextporter
+    volumes:
+      - rextporter-config:/bin/tomlconfig/
+    command: "rextporter -config tomlconfig/main.toml"
+    ports:
+      - "8080:8080"
+    
+  skycoin:
+    image: skycoin/skycoin
+    container_name: skycoin
+    ports:
+      - "6420:6420"
+      - "6000:6000"
+    volumes:
+      - skycoin-data:/data/.skycoin
+      - skycoin-wallet:/wallet
+      
+  
+
+volumes:
+  skycoin-data:
+    external: true
+  skycoin-wallet:
+    external: true
+  rextporter-config:
+    external: true
+```
+## Steps guide
+### 1- Build `rextporter` image
+```
+docker build -t rextporter /path/to/dockerfile
```

Comment:
@mauricio1802 
as this is part of the project, you can reaplace `/path/to/dockerfile` by the real path, a makefile rule could be great too